src/lib.rs

```diff
@@ -9,6 +9,8 @@
 #[cfg(test)]
 extern crate core;
 
+extern crate rlibc;
```

Comment:
I don't think an `extern crate` is necessary here since you're getting the symbols using `extern "C"`.